c++/src/kj/function.h

```diff
@@ -105,7 +105,7 @@ class Function<Return(Params...)> {
 
   // Make sure people don't accidentally end up wrapping a reference when they meant to return
   // a function.
-  KJ_DISALLOW_COPY(Function);
+  KJ_DISALLOW_ONLY_COPY(Function);
   Function(Function&) = delete;
   Function& operator=(Function&) = delete;
```

Comment:
I wonder if these lines can be removed? And in ConstFunction.

Reply:
I think it's been like 10 years since I wrote these lines so I don't quite remember the motivation but I would guess there was one. The rules around non-const copy constructors are kind of complicated. I'd just leave them be.